script/chart_sync.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# Copyright (c) 2018 Bitnami
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+CHARTS_REPO="andresmgot/charts-1"
+CHART_REPO_PATH="bitnami/kubeapps"
+PROJECT_DIR=`cd "$( dirname "${BASH_SOURCE[0]}" )/.." >/dev/null && pwd`
+KUBEAPPS_CHART_DIR="${PROJECT_DIR}/chart/kubeapps"
+
+source $PROJECT_DIR/script/release_utils.sh
+
+changedVersion() {
+    local currentVersion=$(cat "${KUBEAPPS_CHART_DIR}/Chart.yaml" | grep "version:")
+    local externalVersion=$(curl -s https://raw.githubusercontent.com/${CHARTS_REPO}/master/${CHART_REPO_PATH}/Chart.yaml | grep "version:")
```

Comment:
can't we use `git describe`?

Reply:
Oh sorry, I misread this line, can we use https://charts.bitnami.com/bitnami/index.yaml instead?